src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/ClusterAwareConnectionProxy.java

```diff
@@ -57,11 +57,11 @@
 import com.mysql.cj.util.StringUtils;
 import com.mysql.cj.util.Util;
 
+import javax.net.ssl.SSLException;
```

Comment:
Do we need this package?

Reply:
This was already part of the original code, I just used the IDE's optimize import feature and it moved it up.